models/double_entry_transactions/int_quickbooks__credit_card_pymt_double_entry.sql

```diff
@@ -0,0 +1,56 @@
+/*
+Table that creates a debit record to the associated bank account and a credit record to the specified credit card account.
+*/
+
+--To enable this model, set the using_credit_card_payment_txn variable within your dbt_project.yml file to True.
+{{ config(enabled=var('using_credit_card_payment_txn', False)) }}
+
+with credit_card_payments as (
+    select *
+    from {{ref('stg_quickbooks__credit_card_payment_txn')}}
+    where is_most_recent_record
+),
+
+credit_card_payment_join as (
```

Comment:
should we call this CTE something else if there is no join occurring?

Reply:
Good call out, will rename